BeagleBone/README.md

```diff
@@ -0,0 +1,3 @@
+This folder contains live demos that run on the various BeagleBones.
```

Comment:
All the Black demos would run on the original BeagleBone too. I guess since there aren't so many of them, maybe that's not a big deal?